include/OpenSpaceToolkit/Astrodynamics/EventCondition/RealCondition.hpp

```diff
@@ -91,6 +91,14 @@ class RealCondition : public EventCondition
 
     Real evaluate(const State& state) const;
 
+    /// @brief                  Check if the Event Condition is satisfied based on current state and previous
```

Comment:
If I'm understanding this re-org correctly, `RealCondition` is now meant for non-cyclic real numbers, and `AngularCondition` is meant for cyclic ones.  Wouldn't hurt to change this class description to something like:
```
/// @brief                      An Event Condition that can evaluate each state to a real number. Meant for non-cyclic continuous real numbers. For cyclic numbers (like angles), use AngularCondition.`
```